src/test/java/api/ApiBaseTest.java

```diff
@@ -0,0 +1,13 @@
+package api;
+
+import com.epam.data.ResourcesReader;
+
+import java.util.Properties;
+
+import static com.epam.data.Constants.FILE_PATH;
+
+abstract public class ApiBaseTest {
```

Comment:
I am not sure this ApiBaseTest class is needed in case it has only constants. I would suggest to remove base test class, and create one common class for all API testing related constants. Let's call this class APIConstants.java, for example.

Reply:
Fixed